src/components/Quote.js

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from 'react';
+import useFetch from '../hooks/useFetch';
+import styles from '../styles/Quote/Quote.module.css';
+
+const API_KEY = process.env.REACT_APP_API_NINJAS_API_KEY;
+const QUOTES_URL = 'https://api.api-ninjas.com/v1/quotes';
+
+function Quote() {
+  const options = useMemo(
+    () => ({
+      headers: { 'X-Api-Key': API_KEY },
+      contentType: 'application/json',
+    }),
+    [],
+  );
+
+  const { isLoading, data, error } = useFetch(QUOTES_URL, options);
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  if (error) {
+    return <p>Unable to load quote...</p>;
+  }
+
+  const first = data[0];
+
+  return (
+    <div className={styles.wrapper}>
+      <p className={styles.quote}>{first.quote}</p>
+      <p className={styles.right}>
+        by
+        {' '}
+        <span className={styles.author}>{first.author}</span>
+      </p>
+    </div>
+  );
```

Comment:
- [ ] Great work so far. the `quote` property is declared to be undefined when I fired up your app. I tried to console log the `first` variable on line 27 to see the data inside it but seemed to be undefined. I created .env file and added your key to the file but still declared the `quote` as undefined. Kindly look at it and fix where the error is.

### Error
![app-error](https://github.com/henokkhm/microverse-math-magicians/assets/26597432/04483e0f-ae50-4536-8880-389861b45f2b)

### Error
![env](https://github.com/henokkhm/microverse-math-magicians/assets/26597432/dff74f73-d7b2-43d2-9a8a-efb3b608d398)

### Error
![console](https://github.com/henokkhm/microverse-math-magicians/assets/26597432/30e3de85-8be8-44ed-b1f9-152439cb7905)